.github/workflows/enos-run.yml

```diff
@@ -107,9 +113,16 @@ jobs:
           ENOS_VAR_terraform_plugin_cache_dir: ./support/terraform-plugin-cache
           ENOS_VAR_vault_bundle_path: ./support/vault.zip
         run: |
-          enos scenario destroy --timeout 60m0s --chdir ./enos ${{ matrix.scenario }} arch:amd64 builder:crt edition:oss
-      - name: Output debug information on failure
-        if: ${{ failure() }}
+          enos scenario destroy --timeout 60m0s --chdir ./enos ${{ matrix.scenario }}
+      # Temporarily Disable debug output
+      # - name: Output debug information on failure
+      #   if: ${{ failure() }}
+      #   run: |
+      #     env
+      #     find ./enos -name "scenario.tf" -exec cat {} \;
+      - name: Cleanup Enos runtime directories
+        if: ${{ always() }}
         run: |
-          env
-          find ./enos -name "scenario.tf" -exec cat {} \;
+          rm -rf support
+          rm -rf .enos
```

Comment:
Perhaps it might be easier to `rm -rf enos` and not change the working directory, less we might leave something around that is not in the support or generated directories.

Reply:
`enos` is a git added directory, so I didn't want to delete it.  I've seen the generated files in `/tmp` and in `support` and `.enos`.  This should cover the cleanup.  I will also add `rm -rf /tmp/enos*` to cover the `/tmp` directory